### ISSUE TITLE
[bug] "DuckDuckGo Images" doesn't work.

### ISSUE THREAD

**Issue by throwaway242685** [searxng/searxng#2710]
**Version of SearXNG, commit number if you are using on master branch and stipulate if you forked SearXNG**

 2023.9.2+4f8895c6d

**How did you install SearXNG?**

they're public instances.

**What happened?**

"DuckDuckGo Images" doesn't seem to work.

**How To Reproduce.**

search something like: `!ddi snow`

**Expected behavior.**

you should get results.

**Screenshots & Logs.**

![Screenshot_2023-09-04_09-42-00_1.png](https://github.com/searxng/searxng/assets/85243826/7b716ded-1e79-43b2-bce1-966b88783dae)

- Exception: searx.exceptions.SearxEngineAccessDeniedException
- Percentage: 75
- Parameter: HTTP error 403
- Filename: searx/search/processors/online.py:118
- Function: _send_http_request
- Code: response = req(params['url'], **request_args)

**Additional context.**
.

**Comment by Bnyro** [comment]
Duplicate of #2666

The engines itself is fine, the searxng instance you use is rate limited / blocked by DuckDuckGo however.

Next time, please also show the error from the engine.

**Comment by throwaway242685** [comment]
![Screenshot_2023-09-04_09-42-00_1.png](https://github.com/searxng/searxng/assets/85243826/1067863a-0222-4e3d-8ddd-9470cb34f5ac)

**Errors and exceptions.**
- Exception: searx.exceptions.SearxEngineAccessDeniedException
- Percentage: 75
- Parameter: HTTP error 403
- Filename: searx/search/processors/online.py:118
- Function: _send_http_request
- Code: response = req(params['url'], **request_args)

> The engines itself is fine, the searxng instance you use is rate limited / blocked by DuckDuckGo

@Bnyro can you show me an example of an instance where "DDG Images" is working? all the public instances on searx.space have the same problem.

**Comment by return42** [comment]
To validate if a engine work you can test the engine in a instances started locally by `make run` .. but you are right, if I test `!di foo` I get a `if this error persists, please let us know: ops@duckduckgo.com` response back from ddg .. I reopen this issue :+1:
